bluepyemodel/emodel_pipeline/plotting.py

```diff
@@ -1419,15 +1980,61 @@ def plot_models(
             # reset rcParams which are modified by currentscape
             matplotlib.rcParams.update(matplotlib.rcParamsDefault)
 
-        if plot_if_curve:
-            figures_dir_traces = figures_dir / "traces" / dest_leaf
-            IF_curve(
+        if plot_fi_curve:
+            figures_dir_FI_curves = figures_dir / "FI_curves" / dest_leaf
+            FI_curve(
                 mo,
                 mo.responses,
                 copy.deepcopy(cell_evaluator),
-                figures_dir=figures_dir_traces,
+                figures_dir=figures_dir_FI_curves,
             )
 
+    # outside of for loop because we want to load pickle file only once
+    if plot_IV_curve:
+        figures_dir_IV_curves = figures_dir / "IV_curves" / dest_leaf
+        plot_IV_curves(
+            cell_evaluator,
+            emodels,
+            figures_dir_IV_curves,
+            # maybe we should give efel_settings as an argument of plot_models,
+            # like the other pipeline_settings
+            access_point.pipeline_settings.efel_settings,
+            IV_curve_prot_name,
+            custom_bluepyefe_cells_pklpath=custom_bluepyefe_cells_pklpath,
+        )
+
+    if plot_FI_curve_comparison:
+        figures_dir_FI_curves = figures_dir / "FI_curves" / dest_leaf
+        plot_FI_curves_comparison(
+            cell_evaluator,
+            emodels,
+            figures_dir_FI_curves,
+            FI_curve_prot_name,
+            custom_bluepyefe_cells_pklpath=custom_bluepyefe_cells_pklpath,
+        )
+
+    if plot_phase_plot:
+        figures_dir_phase_plot = figures_dir / "phase_plot" / dest_leaf
+        if phase_plot_settings is None:
+            phase_plot_settings = access_point.pipeline_settings.phase_plot_settings
+        phase_plot(
+            emodels,
+            figures_dir_phase_plot,
+            phase_plot_settings["prot_names"],
+            phase_plot_settings["amplitude"],
+            phase_plot_settings["amp_window"],
+            phase_plot_settings["relative_amp"],
+            custom_bluepyefe_cells_pklpath=custom_bluepyefe_cells_pklpath,
+        )
+
+    if plot_traces_comparison:
+        figures_dir_FI_curves = figures_dir / "traces" / dest_leaf
```

Comment:
reusing the variable name `figures_dir_FI_curves` might be confusing

Reply:
Good catch. I have renamed it `figures_dir_traces` in latest commit.